RedHatInsights/insights-results-aggregator-cleaner
# add clowder app file

# Description

add a clowderapp.yaml file for deployment

Fixes # CCXDEV-4922

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

Run

`bonfire deploy -c ./bonfire-config-devel.yaml  -n ephemeral-15 ccx-data-pipeline`

and check the deployment is successful

## Checklist
